.github/workflows/test.yml

```
@@ -0,0 +1,30 @@
+name: Test
+
+on: ['push', 'pull_request']
```

I think to this test should be triggered by ...

- New commit to default branch (main or master branch)
- Create pull request

```suggestion
on:
  pull_request:
  push:
    branches:
      - main
      - master
```